rero_ils/modules/items/serializers/json.py

```diff
@@ -48,6 +51,34 @@ def post_process_serialize_search(self, results, pid_fetcher):
                 with_subtitle=True
             )
 
+            item = Item(metadata)
```

Comment:
Don't understand this. Is it possible to get the Item like this or do we need to use `item=Item.get_record_by_pid(metadata['pid'])

Reply:
done